test/test-buf.c

```diff
@@ -0,0 +1,47 @@
+/* Copyright libuv contributors. All rights reserved.
+ *
+ * Permission to use, copy, modify, and/or distribute this software for any
+ * purpose with or without fee is hereby granted, provided that the above
+ * copyright notice and this permission notice appear in all copies.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ * WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ * MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ * ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ * WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ * ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+ */
+
+#include "uv.h"
+#include "task.h"
+
+#include <stdint.h>
+
+TEST_IMPL(buf_large) {
+  uv_buf_t buf;
+
+  buf = uv_buf_init(NULL, SIZE_MAX);
+  ASSERT(buf.len == SIZE_MAX);
```

Comment:
This test seems like it should not be able to pass on Win64, since `typeof buf.len == ULONG` and is not capable of representing a value of SIZE_MAX. Indeed, it's failing for me locally, so I'm confused that CI was reported to be passing.

Reply:
This is asserting that `(ULONG) UINT64_MAX` == `UINT64_MAX`, which is clearly false, and thus fails for me locally, as I would expect. Does anyone know why CI isn't catching this?